Library/Homebrew/formula.rb

```diff
@@ -1908,6 +1908,27 @@ def to_hash
     hsh
   end
 
+  # @api private
+  # Generate a hash to be used to install a formula from a JSON file
+  def to_bottle_hash(top_level: true)
```

Comment:
might want to rename given mild confusion with `bottle_hash` method below.

Reply:
Any ideas?

Reply:
I went with `to_recursive_bottle_hash` for now